pkg/tsdb/graphite/graphite.go

```diff
@@ -74,6 +81,17 @@ func (e *GraphiteExecutor) Execute(ctx context.Context, queries tsdb.QuerySlice,
 		return result
 	}
 
+	span, ctx := opentracing.StartSpanFromContext(ctx, "alerting.graphite")
```

Comment:
no need to reference the name of the parent span, as child spans will show as nested spans under their parents.